### PR TITLE
CI: Integrate Danger Scripts

### PR DESCRIPTION
**Pull request by JasonGore**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

Integrate fluent-ui-react's Danger package and scripts for commenting and gating PRs in preparation for repo consolidation.

Creating initial PR in draft state for testing.

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/11941)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,45 +8,47 @@
...
     "update-snapshots": "lerna run update-snapshots --stream",
     "update-a11y": "cd apps/a11y-tests && npm run update-snapshots",
-    "check-for-changed-files": "cd scripts && just-scripts check-for-modified-files",
-    "prettier": "node scripts/prettier.js",
-    "generate-version-files": "yarn workspace @uifabric/build just generate-version-files",
-    "codepen": "cd packages/office-ui-fabric-react && node ../../scripts/local-codepen.js",
-    "dom-test": "cd apps/dom-tests && just-scripts jest-dom-with-webpack",
-    "lint": "lerna run lint --parallel",
-    "test": "lerna run test --stream"
+    "vrtest": "cd apps && cd vr-tests && npm run screener"
```
Comment (JasonGore) [inline package.json]: I had to alphabetize this file, it's getting out of control a bit. The meaningful change here is:

```
    "perf:test": "yarn lerna run --stream --scope @fluentui/perf-test perf:test",
    "perf:test:base": "yarn lerna run --stream --scope @fluentui/perf-test perf:test:base",
```

Comment (kenotron) [reply package.json]: shouldn't need to do `yarn lerna run` ... just be `lerna run` is enough

Comment (JasonGore) [reply package.json]: Ok, I've removed `perf:test` commands and also `gulp build` and `build:fluent`. Package level perf-test commands will exist for both Fabric and Fluent and won't appear in root `package.json`.

**Comment by JasonGore** [comment]
I'm going to finalize this PR with the danger job commented out. My biggest fear is overwriting perf data for the FUR repo so I will get back on this as soon as I'm in Prague and can work out blob storage / perf data storage with Miro.

**Pull request merged**